docs/4_the-formal-stuff/career-development.md

```diff
@@ -73,7 +73,7 @@ Our engineering ladders have the following 5 axes (categories):
 4. **Pillar**: makes an impact on the whole Pillar
```

Comment:
Pillar mentions left here

Reply:
Updated this section and another one where Pilar was mentioned.